### PR TITLE
Added qemu x86_64 & build qemu image by packer

### PR DESCRIPTION
**Pull request by kingsd041**
- Adjust the structure of the makefile
- Adjust the script name in the `scripts` folder
- Separate the ec2 and qemu scripts into two targets that run independently

### CODE REVIEW

File: openeuler/qemu/openeuler-qeum-x86_64.pkr.hcl [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  iso_checksum     = "none"
+  output_directory = "output_openEuler_qemu"
+  shutdown_command = "echo 'packer' | sudo -S shutdown -P now"
+  disk_size        = "8G"
+  format           = "qcow2"
+  accelerator      = "none"
+  #http_directory    = "path/to/httpdir"
+  ssh_username     = "root"
+  ssh_password     = "openEuler12#$"
+  ssh_timeout      = "20m"
```
Comment (STARRY-S) [inline openeuler/qemu/openeuler-qeum-x86_64.pkr.hcl:28]: I think 20 minutes is too large, 2 minutes is enough.

**Review by STARRY-S — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -7,16 +7,24 @@
 
 .default:
 	@echo "Usage:"
-	@echo "    make prep  -- Prepare AWS base AMI image."
-	@echo "    make build -- Create AMI image from base image by using packer."
+	@echo "    make base-image  -- Prepare AWS base AMI image."
```
Comment (STARRY-S) [inline Makefile:10]: The `make base-image` only shrink the original qcow2 disk to 8G and does not create any `base AMI image`,
and `make aws-image` uploads the shrinked RAW format qcow2 image to AWS S3 bucket and create AMI image from it.

**Review by STARRY-S — CHANGES_REQUESTED** [review]
Please add more information in your Git commit message, and following [this guide](https://gist.github.com/turbo/efb8d57c145e00dc38907f9526b60f17) to format your commit message.

File: Makefile (hunk 2 above)
Comment (kingsd041) [reply Makefile:10]: updated

File: Makefile [hunk 3]
```diff
@@ -7,16 +7,24 @@
...
+	@echo "    make aws-image -- Create AMI image from base image by using packer."
+	@echo "    make qemu-image -- Create qemu image from base image by using packer."
 	@echo "    make clean -- Delete temporary files in 'tmp' folder."
 
-build: .dapper
+base-image:
+	@echo "Building the base image..."
+	./scripts/base-image
```
[the remark this answers is not in the document]
Comment (kingsd041) [reply Makefile:17]: updated

**Review by STARRY-S — APPROVED** [review]
LGTM, thanks for the changes :-)

**Pull request merged**